web_ui/frontend/package.json

```diff
@@ -8,6 +8,8 @@
     "start": "next start",
     "export": "next export",
     "lint": "next lint",
+    "format": "prettier --check --ignore-path .gitignore .",
```

Comment:
Do we want to include `next lint` here or the other way around? if both of them are running eslint

Reply:
I believe ( and just double checked ) that prettier just formats and doesn't do any linting. The linting is run during the build step.